### ISSUE TITLE
OCP GOLD DR - Order VMs for GOLD DR cluster

### ISSUE THREAD

**Issue by wmhutchison** [BCDevOps/developer-experience#1230]
**Describe the issue**
Due to time constraints, business interests no longer want to wait for the hardware that will eventually be freed up from the 3.11 PROD server and instead want to proceed with a virtualized cluster build for GOLD DR. 

**Additional context**
Add any other context, attachments or screenshots

**Definition of done**
- [x] work with Dan Deane to determine max VM size possible since the goal is to build a cluster that is as production-ready as possible. 
- [x] Build out a template SDSF document containing all the required details.
- [x] Hand over the SDSF template to the team so they can pump out the remaining SDSF docs.
- [x] Hand over the completed SDSF docs for iStore ordering
- [x] Confirm receipt of TheHUB tickets matching the ordered servers.

**Comment by wmhutchison** [comment]
Blocked for now, waiting on iStore ordering process to complete and inject suitable TheHUB tickets to move this along.

**Comment by wmhutchison** [comment]
Not likely to complete by end of Sprint 28, Sprint 29 will be a more viable period.

**Comment by wmhutchison** [comment]
Tickets all received, cluster not yet ready for hand-over (getting close) so we're truing up the tickets involved here so that we're sitting on Handover tickets put on Hold.

**Comment by wmhutchison** [comment]
Tickets now on Hold, will close off.
